### PR TITLE
Update markers alignment

### PR DESCRIPTION
**Pull request by dy**
This PR aligns markers by optical center and makes their size more consistent, see [optical-properties](https://github.com/dfcreative/optical-properties).

Probably needs merging to resolve https://github.com/plotly/plotly.js/pull/1550

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ plot.draw()
 ```
 
 # License
-(c) 2015 Mikola Lysenko. MIT License
+(c) 2016 Dima Yv. MIT License
```
Comment (etpinard) [inline README.md:24]: :tada:

File: scatter.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ var createBuffer = require('gl-buffer')
 var pool = require('typedarray-pool')
 var shaders = require('./lib/shaders')
 var snapPoints = require('snap-points-2d')
-var atlas = require('font-atlas-sdf')
+var atlas = require('../font-atlas-sdf')
```
Comment (etpinard) [inline scatter.js]: :question:

**Review by etpinard — COMMENTED** [review]
Thanks for the PR @dfcreative !

I suspect this needs https://github.com/dfcreative/font-atlas-sdf/commit/5a5a6357714ffbb096e501f43f920ab554efc21b ?

File: scatter.js (hunk 2 above)
Comment (dy) [reply scatter.js]: Whoops, fixed

**Comment by etpinard** [comment]
Alright, here are the new baselines using this branch and `font-atlas-sdf@1.3.2`:

https://github.com/plotly/plotly.js/compare/gl2d-scatter-sdf-marker-alignment

**Comment by etpinard** [comment]
Looking good. especially this one:

peek 2017-06-12 11-09

**Comment by etpinard** [comment]
@dfcreative can you publish `v1.3.10` to npm?

**Comment by dy** [comment]
@etpinard done

**Pull request merged**